py-polars/polars/_html.py

```diff
@@ -75,7 +78,7 @@ def __init__(
 
     def write_header(self) -> None:
         """Write the header of an HTML table."""
-        shape = self.df.shape
+        shape: tuple[int, ...] = self.df.shape
```

Comment:
Why does this need a type hint? The `.shape` property should return `tuple[int, int]`

Reply:
on this line, `shape` is inferred to be `tuple[int, int]`. But then, in
```
        if self.series:
            shape = shape[:1]
```
it is reassigned to `tuple[int]`, which `mypy` throws an error for ("expected tuple[int, int], got tuple[int]").

Typing it as `tuple[int, ...]` satisfies both `tuple[int, int]` and `tuple[int]`

Reply:
Ah, that makes sense with the lines below it indeed. Thanks for the explanation.